packages/marked-shaku-code-annotate/src/index.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  getShakuHighlighters,
+  shouldApplyAnnotation,
+} from "shaku-code-annotate";
+import { MarkedExtension, Token, Tokens, marked } from "marked";
+
+export default function markedShakuCodeAnnotate(options = {}): MarkedExtension {
+  return {
+    async: true,
+    async walkTokens(token) {
+      console.log(token);
+
+      if (token.type !== "code") {
+        return;
+      }
+
+      // @ts-ignore
+      const { lang, meta } = getLangAndMeta(token);
+      const code = token.text;
+
+      console.log(lang, meta);
+      if (!shouldApplyAnnotation(meta)) {
```

Comment:
`shouldApplyAnnotation` should NOT be handled by plugins, rather it should be handle by shaku core